taipy/core/_orchestrator/_dispatcher/_job_dispatcher.py

```diff
@@ -93,11 +91,10 @@ def _execute_job(self, job: Job):
 
     def _execute_jobs_synchronously(self):
         while not self.orchestrator.jobs_to_run.empty():
-            with self.lock:
-                try:
-                    job = self.orchestrator.jobs_to_run.get()
-                except Exception:  # In case the last job of the queue has been removed.
-                    self._logger.warning(f"{job.id} is no longer in the list of jobs to run.")
```

Comment:
The `except` here is wrong because in incase the last job has been removed, the `job.id` in the warning will be the previous job id